pennylane/devices/default_qubit.py

```diff
@@ -412,6 +412,35 @@ def _create_basis_state(self, index):
     def state(self):
         return self._flatten(self._pre_rotated_state)
 
+    def density_matrix(self, wires=None):
+        """Return the reduced density matrix given a set of wires to be traced out.
```

Comment:
Just double checking, but should this instead be "Return the reduced density matrix of a given set of wires"?

Reply:
Yes I've forgotten to change it.